lib/parsers/moov_parser.rb

```diff
@@ -64,13 +64,15 @@ def call(io)
         intrinsics: atom_tree,
       )
     else
+      frame_rate = parse_sample_atom(decoder, atom_tree)&.truncate(2)
```

Comment:
On second thoughts, do we even need truncating here? I see the media duration is kept as is. Is there a reason we need two digits for this value? 🤔

Reply:
wherever I saw a frame rate, it was expressed in the form "ii,dd" . So, whenever I was making the calculation of timescale / sample_duration , the result, if both values are integers is of course an integer, which is not correct in our case. If just one of the 2 becomes a decimal, then we have the correct number but in the format "ii,dddddddd" . By rounding that up we would get the wrong frame rate, while instead truncating it was the thing to do, or at least truncation matched the result given back by the format parser with the frame rate shown by normal video players.

Reply:
Yeah, meant just leaving the value as is, without truncating or rounding. Duration is also usually given with _some_ precision, however we return the whole thing.

If you feel truncation makes sense in this case, let's go with that.